adapters/pubmatic/pubmatic.go

```diff
@@ -473,14 +475,19 @@ func (a *PubmaticAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externa
 				errs = append(errs, err)
 			} else if bidExt != nil {
 				typedBid.Seat = openrtb_ext.BidderName(bidExt.Marketplace)
-				typedBid.BidType = getBidType(bidExt)
+				typedBid.BidType = getMediaTypeForBid(&bid)
```

Comment:
Instead of calling `getMediaTypeForBid` twice, can we use the value of `MType` here?